src/main/java/org/tub/vsp/bvwp/data/container/base/PhysicalEffectDataContainer.java

```diff
@@ -66,7 +67,7 @@ public int hashCode() {
         return result;
     }
 
-    public static final record Effect(Double overall, Double induced, Double shifted) {
+    public static final record Effect(Double overall, Optional<Double> induced, Double shifted) {
```

Comment:
Optional wurde eingeführt, um einen Mechanismus für Rückgabewerte von Funktionen zu haben und ist eigentlich nicht dafür gedacht, Typ eines Attributes zu sein: https://blog.joda.org/2014/11/optional-in-java-se-8.html Auch wenn man das technisch hier so machen könnte, entspricht es nicht der Konvention. Ich schlage daher eines der folgenden vor: 

1. Die drei Werte bekommen wieder den Typ `double` - dann ist der default ja sowieso `0.0`. 
2. Wir machen den Typen `Double` und gehen bei der Verwendung damit um, also z.B. `Optional.ofNullable(effect.induced()).orElse(0.)`

Da ich die Data Container als reines Abbild der Informationen der BVWP-Webseite verstehe, habe ich eine Tendenz zu 2. Da ihr aber die einzigen Verwender seid, kann ich auch mit 1. leben. Dann sollte aber sicher gestellt sein, dass wir auch in Zukunft nicht die Information brauchen, dass das Feld nicht gesetzt war.

Reply:
Danke. Das mit dem ofNullable hatte ich nicht gewusst gehabt und daher da immer weiter gebaut und ein echtes Optional draus gemacht. Da war ich also immerhin nah dran. Ist ja wirklich simple, wenn man es weiß ^^
Habe nun 2) genommen und in einen eigenen PR #13 umgesetzt.